src/vs/workbench/contrib/debug/common/debug.ts

```diff
@@ -910,3 +912,9 @@ export interface IDebugHelperService {
 
 	createTelemetryService(configurationService: IConfigurationService, args: string[]): TelemetryService | undefined;
 }
+
+export interface IDebugCompoundRoot {
```

Comment:
I do not see the benefits of introducing this interface.
Though we can have it if you prefer it that way.

Reply:
Yea I originally had the compound root as required and used the stub, but we no longer do that so I'll clean it up!